confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConstant.java

```diff
@@ -27,13 +27,13 @@ public class ConfigNodeConstant {
           ? ConfigNodeConstant.class.getPackage().getImplementationVersion()
           : "UNKNOWN";
 
-  public static final String GLOBAL_NAME = "IoTDB ConfigNode";
+  public static final String GLOBAL_NAME = "IoTDB-ConfigNode";
   public static final String CONFIGNODE_CONF = "CONFIGNODE_CONF";
   public static final String CONFIGNODE_HOME = "CONFIGNODE_HOME";
 
   public static final String ENV_FILE_NAME = "confignode-env";
-  public static final String CONF_NAME = "iotdb-confignode.properties";
-  public static final String SPECIAL_CONF_NAME = "iotdb-confignode-special.properties";
+  public static final String CONF_FILE_NAME = "iotdb-confignode.properties";
+  public static final String SYSTEM_FILE_NAME = "confignode-system.properties";
```

Comment:
Why does one have the `iotdb` prefix and the other not?

Reply:
Named by Mr.Qiao~

Reply:
ok